apps/src/music/views/Timeline.tsx

```diff
@@ -22,13 +22,13 @@ const paddingOffset = 10;
 // Start scrolling the playhead when it's more than this percentage of the way across the timeline area.
 const playheadScrollThreshold = 0.75;
 
-const getEventHeight = (numUniqueRows: number, availableHeight = 110) => {
+const getEventHeight = (numUniqueRows: number, availableHeight = 130) => {
   // While we might not actually have this many rows to show,
   // we will limit each row's height to the size that would allow
   // this many to be shown at once.
   const minVisible = 5;
 
-  const maxVisible = 10;
+  const maxVisible = 26;
```

Comment:
I've tried it from time to time, but it would require a deeper dive.  

Some issues to consider include:

- calculating the timeline height in this class rather than in `TimelineSimple2Events`.
- testing across all browsers to see how it feels; some browsers lock the primaxy axis once you start scrolling in one, which hasn't always matched user intent.
- deciding whether we want to be consistent with Blockly's two-axis scrolling which allows the user to drag the workspace as well as scroll it natively.
- deciding on the right experience when playback begins and we auto-scroll the timeline horizontally.  It might be better to intelligently adjust vertical scroll in that case rather than, or in addition to, allowing the user to vertically scroll.
- whether users are likely to miss events that appear in corners; an advantage of scrolling in one axis only is that a full sweep back and forth can reveal everything.  

That said, this is best explored practically, so I might separately prepare a small URL parameter that lets us start playing with vertical scrolling.

Reply:
Interesting and- thanks for sharing the many complex considerations!